bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -1636,4 +1642,23 @@ private void postApprovalNeededComment() {
         pr.addComment(message);
     }
 
+    /**
+     * Iterate through each entry in the regularIssuesMap. If the value is present, get all CSR links associated with this issue.
+     * "all CSR links" means that the CSR linked with the primary issue and the CSRs linked with any backport issues of the primary issue.
+     * Constructs a map from main issue ID to all CSR links.
+     */
+    private Map<String, List<Link>> issueToCsrLinksMap(Map<Issue, Optional<IssueTrackerIssue>> regularIssuesMap) {
```

Comment:
Looking at this some more, I think it would make more sense if we made the map from issue ID to `List<IssueTrackerIssue>`. Both consumers of this map map to the linked Issue anyway.
```suggestion
    private Map<String, List<IssueTrackerIssue>> issueToAllCsrsMap(Map<Issue, Optional<IssueTrackerIssue>> regularIssuesMap) {
```

Reply:
Make sense.  Will change it